### PR TITLE
Addition of hydrophone delta message

### PR DESCRIPTION
**Pull request by ryan-summers**
Adds in the hydrophone delta message as defined in http://robosub.eecs.wsu.edu/wiki/cs/topics_list/start#hydrophones

### CODE REVIEW

File: msg/HydrophoneDeltas.msg
Comment (skallaher) [inline msg/HydrophoneDeltas.msg]: I feel like these could be named to better represent which hydrophone deltas they are. (e.g. xDelta, yDelta, zDelta) I'm also assuming you're using these as the time deltas from the reference hydrophone, correct?

**Review by skallaher — DISMISSED** [review]

File: msg/HydrophoneDeltas.msg
Comment (ryan-summers) [reply msg/HydrophoneDeltas.msg]: Correct. I was following James's specification, but I agree that your proposed naming conventions are more intuitive and understandable.

Comment (skallaher) [reply msg/HydrophoneDeltas.msg]: @irwineffect what are your thoughts?

**Comment by skallaher** [comment]
I will dismiss my review so that I don't block merging, though I recommend that @irwineffect give this a look to approve the naming change. Otherwise, looks good!

File: msg/HydrophoneDeltas.msg
Comment (irwineffect) [reply msg/HydrophoneDeltas.msg]: Yeah I agree with xDelta, yDelta, zDelta

**Review by irwineffect — APPROVED** [review]
Looks good!

**Pull request merged**
